### ISSUE TITLE
Fix SybaseIQDatabaseConnector

### ISSUE THREAD

**Issue by shantstepanian** [schemacrawler/SchemaCrawler-Database-Plugins#2]
Hi Sualeh, long time! Hope all is well.

I finally upgraded my [Obevo](https://github.com/goldmansachs/obevo) project to Java 8, and so I'm upgrading my dependency to the latest SchemaCrawler version. Thus, the SchemaCrawler-Java7-Backport and SchemaCrawler-Sybase projects no longer need maintenance, and I see you've already archived them.

That said, I'd like to have the SchemaCrawler-Sybase module brought into the main SchemaCrawler project. Hopefully won't be too difficult to do from your side; I'd send a PR, but I saw your note on PRs

A note on the existing files in https://github.com/schemacrawler/SchemaCrawler-Sybase - the SybaseIQDatabaseConnector.java file had a bug; the line for databaseSpecificOverrideOptionsBuilder.doesNotSupportCatalogs() does not belong in this file; it should only be in the Odbc connector

I'd expect the SybaseIQDatabaseConnector class to look something like below
```
public final class SybaseIQDatabaseConnector extends DatabaseConnector {
    private static final long serialVersionUID = 1L;

    public SybaseIQDatabaseConnector() throws IOException {
        super(new DatabaseServerType("sybaseiq", "SAP Sybase IQ"),
                new ClasspathInputResource("/schemacrawler-sybaseiq.config.properties"),
                (informationSchemaViewsBuilder, connection) -> informationSchemaViewsBuilder.fromResourceFolder("/sybaseiqodbc.information_schema")
        );
    }

    @Override
    protected Predicate<String> supportsUrlPredicate() {
        // always return false here as this conflicts with SAP Sybase ASE
        return url -> false;
    }
}
```

and the ODBC driver would have this extra method:

```
    @Override
    public SchemaRetrievalOptionsBuilder getSchemaRetrievalOptionsBuilder(Connection connection) {
        return super.getSchemaRetrievalOptionsBuilder(connection)
                .withDoesNotSupportCatalogs();  // Unlike the regular JDBC driver, catalogs are not supported
    }
```

Please let me know if you need more information

**Comment by sualeh** [comment]
@shantstepanian - good to hear from you. The Java 8 version of the SybaseIQDatabaseConnector in https://github.com/schemacrawler/SchemaCrawler-Database-Plugins, so you can submit a pull request here. I will be happy to accept your pull request in this project.

**Comment by shantstepanian** [comment]
Hi Sualeh,
Would it be possible for you to make the change on my behalf? I cannot PR at the moment

The only change - in [this file](https://github.com/schemacrawler/SchemaCrawler-Database-Plugins/blob/master/schemacrawler-sapiq/src/main/java/schemacrawler/server/sapiq/SAPIQDatabaseConnector.java), to remove the method "getSchemaRetrievalOptionsBuilder"; the base implementation suffices for that

Regarding this database-plugins project, do you always release in lockstep w/ core schemacrawler? I ask as I want see how to handle the version numbering in my poms

**Comment by sualeh** [comment]
@shantstepanian - the jars from this project will always have the same version number as the main project, and will be released together.

**Comment by sualeh** [comment]
@shantstepanian  - I have released [SchemaCrawler 16.9.1](https://search.maven.org/search?q=us.fatehi%20schemacrawler) with this fix.
